Dockerfile

```
@@ -0,0 +1,9 @@
+# Inherit from an upstream image
+FROM jupyter/base-notebook:2023-05-15
```

```suggestion
FROM pangeo/base-image:2023.05.08
```
what are the benefits for switching to jupyter relative to the pangeo base-image?